docs/pages/template_schema/companion_relationships.rst

```diff
@@ -0,0 +1,13 @@
+Companion Relationships
+=========================
+
+.. mdinclude:: https://github.com/astrodbtoolkit/astrodb-template-db/blob/main/docs/schema/CompanionParameters.md
+
+
+Notes
+-----
+Relationships are constrained but should be one of the following:
```

Comment:
Typo? `Relationships are unconstrained but ...` or `Relationships should be constrained to be one ...`